tool-base/src/main/java/io/spine/tools/java/fs/SourceFile.java

```diff
@@ -194,16 +224,26 @@ private static IllegalStateException invalidNestedDefinition(String filename,
      *         the file descriptor containing the enum descriptor
      * @return the relative file path
      */
-    @SuppressWarnings("unused") /* Part of the public API. */
+    public static SourceFile forEnum(EnumDescriptor enumType, FileDescriptor file) {
+        return forEnum(enumType.toProto(), file.toProto());
```

Comment:
Please check the arguments for `null`s. 

Also, a `NullPointerTester` should be probably applied here to test public static API like this one.